### ISSUE TITLE
nueral_network forward propagation no activation

### ISSUE THREAD

**Issue by yummyyummya** [aymericdamien/TensorFlow-Examples#262]
in neural_network_raw, the forward propagation function neural_net(x), layer_1 = Wx+b, but there is no activation sigmoid(layer_1), this will bring some problems.

**Comment by vishalmhjn** [comment]
yes, it is surprising how can this NN work without activation functions

**Comment by DCNick3** [comment]
Well, its accuracy is same as logistic regression. Adding activation makes it much more accurate.
